demo/cyberphys/main_fett.c

```diff
@@ -274,16 +274,16 @@ void main_fett(void)
         vTaskSuspendAll();
     }
 
-    FreeRTOS_printf(("\n>>>Beginning of Fett<<<\r\n"));
+    FreeRTOS_printf(("\n>>>Beginning of Besspin<<<\r\n"));
     
     startNetwork();
 
     funcReturn = xTaskCreate(prvMainTask, "prvMainTask", MAINTASK_STACK_SIZE, NULL, MAINTASK_PRIORITY, NULL);
 
     if (funcReturn == pdPASS) {
-        FreeRTOS_printf(("%s (Info)~  main_fett: Created prvMainTask successfully.<<<\r\n", getCurrTime()));
+        FreeRTOS_printf(("%s (Info)~  main_besspin: Created prvMainTask successfully.<<<\r\n", getCurrTime()));
     } else {
-        FreeRTOS_printf(("%s (Error)~  main_fett: Failed to create prvMainTask.<<<\r\n", getCurrTime()));
+        FreeRTOS_printf(("%s (Error)~  main_besspin: Failed to create prvMainTask.<<<\r\n", getCurrTime()));
```

Comment:
These seem wrong when it's still main_fett.c and the function is called main_fett

Reply:
I wanted to do just the minimal changes so the program can be booted in besspin-tool. The proper fix would be to either integrate the CHERI changes into our fork, or update the CHERI demos with the new cyberphys demo.

Reply:
The main change being renaming `*fertt` to `*besspin`